Jenkinsfile

```diff
@@ -193,6 +193,12 @@ node('ubuntu-zion-legacy') {
             OsTools.runSafe(this, "docker tag ${slimImageId} ${organization}/${dockerHubRepository}:${version}-slim")
             OsTools.runSafe(this, "docker tag ${slimImageId} ${organization}/${dockerHubRepository}:latest-slim")
 
+            // push the images
+            OsTools.runSafe(this, "docker image push ${organization}/${dockerHubRepository}:${version}")
+            OsTools.runSafe(this, "docker image push ${organization}/${dockerHubRepository}:latest")
+            OsTools.runSafe(this, "docker image push ${organization}/${dockerHubRepository}:${version}-slim")
+            OsTools.runSafe(this, "docker image push ${organization}/${dockerHubRepository}:latest-slim")
+
```

Comment:
this needs to move down into the withCredentials block below, since DOCKER_CONTENT_TRUST=1 is set above. these will try to sign as you push with that set.

Reply:
a46ed42cb6f73ad2827d8e818f1af7ca05129bb0